documentation/src/docbkx/chapter22-improve.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <chapter xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	id="improve" xmlns="http://www.oasis-open.org/docbook/4.5" xmlns:xl="http://www.w3.org/1999/xlink"
```

Comment:
Err... Eeeehhhmm... Yeah. I misunderstood the purpose of "id" in chapter... I actually thought I put it back, but obviously not.